web/src/main/webapp/WEB-INF/data/data/resources/config/manual.json

```diff
@@ -1,38 +1,38 @@
-  "installing-db": "install-guide/configuring-database.html",
-  "installing-index": "install-guide/installing-index.html",
-  "standards-home": "annexes/standards/index.html"
+  "creating-metadata": "user-guide/describing-information/creating-metadata/",
+  "metadata-identifier-template": "administrator-guide/managing-metadata-standards/metadata-identifier/",
+  "implementing-a-schema-plugin": "customizing-application/implementing-a-schema-plugin/",
+  "quick-start-home": "user-guide/quick-start/index/",
+  "batchupdate": "guide/workflow/batchupdate/",
+  "doi": "user-guide/associating-resources/doi/",
+  "life-cycle": "user-guide/workflow/life-cycle/",
+  "tagging-with-categories": "user-guide/tag-information/tagging-with-categories/",
+  "publishing-home": "user-guide/publishing/index/",
+  "geopublication": "user-guide/workflow/geopublication/",
+  "linking-documents": "user-guide/associating-resources/linking-documents/",
+  "linking-dataset-or-service": "user-guide/associating-resources/linking-dataset-or-service/",
+  "linking-records": "user-guide/associating-resources/linking-records/",
+  "linking-others": "user-guide/associating-resources/linking-others/",
+  "associating-resources-home": "user-guide/associating-resources/index/",
+  "harvesting-home": "user-guide/harvesting/index/",
+  "batchediting": "user-guide/workflow/batchediting/",
+  "importing-metadata": "user-guide/describing-information/importing-metadata/",
+  "map-server-configuration": "administrator-guide/configuring-the-catalog/map-server-configuration/",
+  "managing-categories": "administrator-guide/managing-classification-systems/managing-categories/",
+  "managing-thesaurus": "administrator-guide/managing-classification-systems/managing-thesaurus/",
+  "csw-configuration": "administrator-guide/configuring-the-catalog/csw-configuration/",
+  "portal-configuration": "administrator-guide/configuring-the-catalog/portal-configuration/",
+  "system-configuration": "administrator-guide/configuring-the-catalog/system-configuration/",
+  "inspire-configuration": "administrator-guide/configuring-the-catalog/inspire-configuration/",
+  "user-self-registration": "administrator-guide/managing-users-and-groups/user-self-registration/",
+  "user-interface-configuration": "administrator-guide/configuring-the-catalog/user-interface-configuration/",
+  "creating-group": "administrator-guide/managing-users-and-groups/creating-group/",
+  "creating-user": "administrator-guide/managing-users-and-groups/creating-user/",
+  "managing-directories": "administrator-guide/managing-classification-systems/managing-directories/",
+  "installing-index": "install-guide/installing-index/",
+  "maintainer-guide-home": "maintainer-guide/index/",
+  "installing-datadirectory": "install-guide/customizing-data-directory/",
+  "installing-db": "install-guide/configuring-database/",
+  "installing-index": "install-guide/installing-index/",
+  "standards-home": "annexes/standards/index/"
```

Comment:
maintaining things in two places is difficult; I wonder in the future if this could be generated from the mkdocs.yml file?

Reply:
Maybe can be generated automatically, we extracted the links used in GeoNetwork to a file as previously the links were hardcoded in the code.